### PR TITLE
start of rework of overview

### PR DESCRIPTION
**Pull request by epsilonhalbe**
with some hardcoded in one of the dropdown menuse, as it took me some time to get those dropdown menus to work properly.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Workflow for deployment
 > cd path/to/gonimo-front
 > bower update
 > rm -rf ./bower_components/purescript-newtype
-> pulp browserify --to dist/app.js
+> pulp browserify --to dist/app.js && cp -R ./static/* dist
```
Comment (eskimor) [inline Readme.md:25]: is the cp static stuff still relevant?

File: support/index.html [hunk 2]
```diff
@@ -5,12 +5,16 @@
     <meta http-equiv="Content-type" content="text/html; charset=utf-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>Pux Starter App</title>
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
+    <link rel="stylesheet" href="./bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
```
Comment (eskimor) [inline support/index.html:8]: Warum verwenden wir jetzt eine lokale Version?

File: webpack.config.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ var path = require('path');
 var webpack = require('webpack');
 var HtmlWebpackPlugin = require('html-webpack-plugin');
 
-var port = process.env.PORT || 3000;
+var port = process.env.PORT || 3001;
```
Comment (eskimor) [inline webpack.config.js:5]: Is this port change on purpose?

**Comment by eskimor** [comment]
Thanks a lot @epsilonhalbe!

**Comment by epsilonhalbe** [comment]
Yes you are right it is not relevant, in fact it overwrites the index.html

On 27 October 2016 07:46:15 CEST, Robert Klotzner notifications@github.com wrote:

> eskimor commented on this pull request.
> 
> > @@ -22,7 +22,7 @@ Workflow for deployment
> > cd path/to/gonimo-front
> > bower update
> > rm -rf ./bower_components/purescript-newtype
> > -> pulp browserify --to dist/app.js
> > +> pulp browserify --to dist/app.js && cp -R ./static/\* dist
> 
> is the cp static stuff still relevant?
> 
> ## 
> 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/gonimo/gonimo-front/pull/11#pullrequestreview-5994096

## 

Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Pull request merged**